csvReader.js

```diff
@@ -0,0 +1,16 @@
+const { createReadStream, createWriteStream } = require('node:fs');
+const { pipeline } = require('node:stream/promises');
+const csv = require('csvtojson');
+
+class CsvReader {
+    async read(readFilePath, writeFilePath = './output.txt') {
+
+        await pipeline(
```

Comment:
So, if you use promisified version of `pipeline` here, I guess you need to add `try/catch` here to handle an error, or use simple version of `pipeline` and callback as a 4th argument

Reply:
done